doctor/models.py

```
@@ -0,0 +1,595 @@
+        verbose_name='ساعت شروع'
+    )
+    end_time = models.TimeField(
+        verbose_name='ساعت پایان'
+    )
+    
+    # نوع ویزیت
+    VISIT_TYPE_CHOICES = [
+        ('in_person', 'حضوری'),
+        ('online', 'آنلاین'),
+        ('both', 'هر دو'),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)